__init__.py

```diff
@@ -355,10 +362,15 @@ def after_load(_a, _b):
 
 
 def gameEditorUpdate(self, context):
+    world_defaults = {}
     if self.gameEditorMode == "SM64":
-        self.f3d_type = "F3D"
+        self.f3d_type, world_defaults = "F3D", sm64_world_defaults
     elif self.gameEditorMode == "OOT":
-        self.f3d_type = "F3DEX2/LX2"
+        self.f3d_type, world_defaults = "F3DEX2/LX2", oot_world_defaults
+    elif self.gameEditorMode == "Homebrew":
+        self.f3d_type, world_defaults = "F3D", {}  # This will set some pretty bad defaults, but trust the user
```

Comment:
nit: I don't really like doing ``var_a, var_b = value_1, value_2``

Reply:
I do this relatively often so I'd like to ask why? I assume its a redability thing but I personally can easily following what's going on

Reply:
I'm not really a Python dev so can't speak on general consensus here, but in general code should be as readable as possible, as much as possible. Just because one line is easily understood by itself doesn't mean it won't contribute to an overwhelming function if these shortcuts are taken all over the place.